### ISSUE TITLE
Attempting to delete default title sequences may crash game

### ISSUE THREAD

**Issue by Wuzzy2** [OpenRCT2/OpenRCT2#4034]
**OS:** GNU/Linux (Arch Linux)
**Version:** 0.0.5-develop
**Commit/Build:** 59c3656
- Reproducible in RCT2 (vanilla)? N/A
- Multiplayer? N/A

**Steps to reproduce:**
1. Make sure you have selected one of the default title sequences (e.g. OpenRCT2) selected
2. Restart game
3. Stay in the main menu
4. In the options, open the title sequence window, select the current title sequence and try to delete it

Expected: OpenRCT2 gives an error message telling me that this is not allowed.
Actual result: OpenRCT2 immediately crashes after doing this.

Now try the same steps, but instead, try to delete a different default title sequence which is _not_ currently running in the main menu.

Expected: OpenRCT2 gives an error message telling me that this is not allowed.
Actual result: The title sequence will be removed from the list. Luckily, it is restored after restarting the game.

So yeah, the title sequence window is a bit buggy at the moment.

Edit: For some reason, only the OpenRCT2, RCT2 and RCT + AA + LL title sequences can be “fake deleted”, there is actually an error message when you try to remove RCT or RCT + AA.

**Comment by mrtnptrs** [comment]
I can confirm that deleting the RCT and RCT + AA results in the correct error mesages. The other ones can just be deleted.

**Comment by mrtnptrs** [comment]
When I look at the console while deleting them, I see this:

```
ERROR[src\interface\title_sequences.c:186 (title_sequence_delete_preset)]: Failed to delete directory: "C:\Users\maart\Documents\OpenRCT2\title sequences\RollerCoaster Tycoon 1 (AA + LL)"
ERROR[src\interface\title_sequences.c:186 (title_sequence_delete_preset)]: Failed to delete directory: "C:\Users\maart\Documents\OpenRCT2\title sequences\RollerCoaster Tycoon 2"
ERROR[src\interface\title_sequences.c:186 (title_sequence_delete_preset)]: Failed to delete directory: "C:\Users\maart\Documents\OpenRCT2\title sequences\OpenRCT2"
```
So, the game sais that they are deleted and they come back after a restart, but the console sais that it failed to delete the files/directories.

**Comment by rwjuk** [comment]
Fixed in #6592 - an error is now correctly displayed when a user attempts to deleted a built-in title sequence.
